src/Sarif/Visitors/GitHubDspIngestionVisitor.cs

```diff
@@ -43,32 +55,103 @@ public override Run VisitRun(Run node)
                             errors.Add(result);
                         }
 
-                        // GitHub DSP reportedly has an ingestion limit of 500 issues
-                        if (errors.Count > 500) { break; }
+                        if (errors.Count > s_MaxResults) { break; }
```

Comment:
>> [](start = 41, length = 1)

== instead of > ?

Reply:
Right :)

---
In reply to: [483744382](https://github.com/microsoft/sarif-sdk/pull/2061#discussion_r483744382) [](ancestors = 483744382)